astropy/utils/masked/function_helpers.py

```diff
@@ -188,18 +188,22 @@
 dispatched_function = FunctionAssigner(DISPATCHED_FUNCTIONS)
 
 
-def _get_data_and_masks(*args):
+def _get_data_and_mask(array):
```

Comment:
Optional, but maybe naming this `_get_data_and_mask_as_array` might help make the intent clear in the code that calls this.

Reply:
Yes, good suggestion -- as is, it is just confusing that the name is the same with just an underscore added.  I made it `_get_data_and_mask_array` (saving the plural for the other routine).